stingray/io.py

```diff
@@ -680,7 +687,7 @@ def _save_ascii_object(object, filename):
 def _retrieve_ascii_object(object, filename):
     pass
 
-def write(input, filename, format = 'pickle'):
+def write(input, filename, format = 'pickle', save_as_dict = True):
```

Comment:
Yes, I agree. Though we should make sure we have good documentation for that, and not go the matplotlib route where one never knows what actually goes into the function ... :)